core/src/builder/companion/src/expand/mod.rs

```diff
@@ -86,8 +86,8 @@ pub fn expand(def: CompanionDef) -> SynResult<TokenStream> {
 				#(
 					let para = _hidden_FudgeParaChain {
 						id: _hidden_ParaId::from(#parachain_ids),
-						head: companion.#parachain_names.head(),
-						code: companion.#parachain_names.code(),
+						head: companion.#parachain_names.head().map_err(|_| ())?,
+						code: companion.#parachain_names.code().map_err(|_| ())?,
```

Comment:
What do you think about forwarding the error here? Wouldn't that be possible?

Reply:
Done - I did not use the specific parachain/relaychain/standalone error types to avoid having to deal with generic arguments for `BuilderError` which in turn complicates the macro usage by a small degree.